lib/ansible/plugins/connection/winrm.py

```diff
@@ -215,6 +214,7 @@ def _connect(self):
 
     def exec_command(self, cmd, in_data=None, sudoable=True):
         super(Connection, self).exec_command(cmd, in_data=in_data, sudoable=sudoable)
+        cmd = self._shell.join_cmd_list(cmd)
         cmd_parts = shlex.split(to_bytes(cmd), posix=False)
         cmd_parts = map(to_unicode, cmd_parts)
```

Comment:
The added join here is a bit misleading. It's there to maintain consistency with other the plugins. We always pass cmd as a list to exec_command now, but in the case of powershell that list always contains a single item -- the cmd that was previously passed as a string.

Take a look at the changes to [powershell.py](https://github.com/ansible/ansible/pull/14845/files#diff-9b6915232549a7bcd00b8b8f142b7232) and I think you'll see what I mean.